### DISCUSSION TITLE
Could not update templates: could not read configuration file: EOF

### DISCUSSION THREAD

**Discussion by cyb3rzest** [projectdiscovery/nuclei#3638] [Q&A] [answered]
Issue in running nuclei, have reinstalled the engine as well as template 3 times, still the issue persist.

### Command Given or Steps To Reproduce:
nuclei -target example.com -t /root/nuclei-templates
nuclei -ut
nuclei -update-template

### Nuclei version:
 issue in latest version

### Current Behavior:
[ERR] Could not update templates: could not read configuration file: EOF
![image](https://user-images.githubusercontent.com/29830064/223462328-5ce1aca0-ea04-4f5f-9ff2-71b05cad9747.png)

### Expected Behavior:
It should have updated the same, as well as started to search for the vulns in the target domain, but it freezes after giving the command

Kindly fix it

**Comment by RamanaReddy0M** [comment, marked as answer]
@cyb3rzest, could you please try updating Nuclei to the latest version (v2.9.2)?

I tested with the latest version and didn't notice any problems.

```console
nuclei -target example.com -t ~/nuclei-templates

                     __     _
   ____  __  _______/ /__  (_)
  / __ \/ / / / ___/ / _ \/ /
 / / / / /_/ / /__/ /  __/ /
/_/ /_/\__,_/\___/_/\___/_/   v2.9.2

		projectdiscovery.io

[INF] Current nuclei version: v2.9.2 (latest)
[INF] Current nuclei-templates version: 9.4.3 (latest)
[INF] New templates added in latest release: 55
[INF] Templates loaded for current scan: 5900
[INF] Targets loaded for current scan: 1
[INF] Running httpx on input host
[INF] Found 1 URL from httpx
[INF] Templates clustered: 1052 (Reduced 973 Requests)
[http-missing-security-headers:content-security-policy] [http] [info] https://example.com
[http-missing-security-headers:permissions-policy] [http] [info] https://example.com
[http-missing-security-headers:cross-origin-resource-policy] [http] [info] https://example.com
[http-missing-security-headers:access-control-allow-credentials] [http] [info] https://example.com
.....
```

Let me know if the problem still persists, thank you.
